src/rules/no_irregular_whitespace.rs

```diff
@@ -175,149 +219,174 @@ mod tests {
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{000C} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{00A0} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{feff} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{2000} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{2001} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{2002} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{2003} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{2004} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{2005} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{2006} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{2007} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{2008} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
       ],
       "var any \u{2009} = 'thing';": [
         {
           col: 8,
           message: NoIrregularWhitespaceMessage::NotAllowed,
+          hint: HINT,
         }
-      ],
+        ],
```

Comment:
```suggestion
      ],
```

Looks like more indent than it should have